src/interfaces/correctionFiling-interface.ts

```diff
@@ -0,0 +1,25 @@
+import { IncorporationApplicationIF } from '@/interfaces/incorporation-interfaces'
+
+/** Incorporation Application filing loaded from / saved to the Legal API. */
+export interface CorrectionFilingIF {
+  header: {
+    name: string
+    certifiedBy: string
+    date: string
+    routingSlipNumber?: string
+    folioNumber?: string
+    effectiveDate?: string
```

Comment:
I think you're missing some fields. See Vysakh's PR for all the header fields.

Reply:
Copy that, will do!

Reply:
Updated in next commit